partials/anunt.hbs

```diff
@@ -0,0 +1,75 @@
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+
+<!-- Carousel -->
+<style>
+    .container-carousel {
+        width: 100%;
+        background-color: white;
+        {{!-- background-color: white; --}}
+    }
+
+    h1.carousel-title {
+        font-size: 2rem;
+        text-align: center;
+    }
+
+    h2.carousel-subtitle{
+        font-size: 1.1rem;
+        text-align: center;
+    }
+</style>
+
+<div class="container-carousel m-0 p-5">
+
+    {{#if title}}
+    <h1 class="carousel-title m-0">
+        {{title}}
+    </h1>
+    {{/if}}
+
+    {{#if subtitle}}
+    <h2 class="carousel-subtitle m-2 mb-5">
+        {{subtitle}}
+    </h2>
+    {{/if}}
+
+    {{!-- Carousel --}}
+
+    <div id="anunturi" class="carousel slide" data-bs-ride="carousel">
+
+        <!-- Indicators/dots -->
+        <div class="carousel-indicators">
+            <button type="button" data-bs-target="#anunturi" data-bs-slide-to="0" class="active"></button>
+            <button type="button" data-bs-target="#anunturi" data-bs-slide-to="1"></button>
+            <button type="button" data-bs-target="#anunturi" data-bs-slide-to="2"></button>
+            <button type="button" data-bs-target="#anunturi" data-bs-slide-to="3"></button>
+
+            {{!-- {{#for 0 10 2}} 
+            <button type="button" data-bs-target="#anunturi" data-bs-slide-to="{{this}}"></button>
+            {{/for}} --}}
+            {{!-- <button type="button" data-bs-target="#anunturi" data-bs-slide-to="4"></button> --}}
+        </div>
+
+        <div class="carousel-inner">
+            {{!-- .carousel-item.active*10>img[src="https://picsum.photos/id/$/1920/1080" class="w-100 h-50" alt="Imagine $"] --}}
+            <div class="carousel-item active"><img src="https://picsum.photos/id/1/1920/1080" alt="Imagine 1" class="w-100 h-50"></div>
```

Comment:
Here instead of photos we need to filter the posts from osut with the tag "important". You can find an example for this in page-proiecte.hbs starting from line 46.  Limit them to 3 per "slide".
Also check if the posts are spaced well enough , something like this:
![image](https://user-images.githubusercontent.com/57523010/224951966-e62bb46d-6c22-4b79-8146-4c1a93a5e38e.png)